WordPress/Classes/ViewRelated/Post/PostCardTableViewCell.m

```diff
@@ -359,20 +359,15 @@ - (void)configureMetaButtons
     [self resetMetaButton:self.metaButtonRight];
     [self resetMetaButton:self.metaButtonLeft];
 
-    // We don't have comment and like counts for self-hosted sites.
-    if (![self.contentProvider isWPcom]) {
-        return;
-    }
-
     NSMutableArray *mButtons = [NSMutableArray arrayWithObjects:self.metaButtonLeft, self.metaButtonRight, nil];
-    if ([self.contentProvider numberOfComments] > 0) {
+    if ([self.contentProvider supportsComments] && [self.contentProvider numberOfComments] > 0) {
```

Comment:
:shrug: maybe? Seems like its just adding noise if there is no real purpose.  If the next dev who comes along would wonder why both checks were being made maybe we don't need it.